src/ruby.ts

```diff
@@ -35,6 +35,9 @@ export class Ruby {
         case "rvm":
           await this.activate("rvm-auto-ruby");
           break;
+        case "none":
```

Comment:
🤔 @vinistock I wonder if it's more coherent to make this the default, even though it means an additional step is needed for Shopify staff using `shadowenv`.

Reply:
I agree that `shadowenv` is not the best default for most users in the community.

But this will likely need a coordinated change with Spin/dev to make sure we don't break Shopify devs' development experience?

Reply:
IMO the default should be "auto" where we try to discover the version manager (or the lack of it) automatically based on some search order that starts with "shadowenv" and has "none" as the ultimate fallback. If this doesn't work for specific setups, folks can still explicitly set their version manager.